app/assets/javascripts/homePage.js

```diff
@@ -4,5 +4,6 @@
 //= require_tree ./shared
 //= require_tree ./sidebar
 //= require_tree ./profile
+//= require_tree ./homepage
 
-app = angular.module("HomePage", ["ui.bootstrap", "ngRoute", "Profile", "Sidebar"])
+app = angular.module("HomePage", ["ui.bootstrap", "ngRoute", "SharedServices", "Profile", "Sidebar", "Conversation"])
```

Comment:
The SharedServices has been removed. However, the require_tree ./homepage is necessary to include the conversation script.